avocado/plugins/assets.py

```diff
@@ -24,19 +24,25 @@
 from avocado.core import safeloader
 from avocado.core.output import LOG_UI
 from avocado.core.plugin_interfaces import CLICmd
+from avocado.core.plugin_interfaces import JobPreTests
 from avocado.utils.asset import Asset
 from avocado.utils import data_structures
 
 
-class FetchAssetHandler(ast.NodeVisitor):
+class FetchAssetHandler(ast.NodeVisitor):  # pylint: disable=R0902
     """
     Handles the parsing of instrumented tests for `fetch_asset` statements.
     """
 
     PATTERN = 'fetch_asset'
 
-    def __init__(self, file_name):
+    def __init__(self, file_name, klass=None, method=None):
```

Comment:
I would suggest to follow PEP8:

"If a function argument's name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.)"

Reply:
No, `klass` is really a subterfuge for `class`.

We have a lot of those uses in Avocado (144), and I'm probably the one to blame for most if not all of them.  I think it'd be quite hard to have a pylint check for that, so the best we can do is  a mass change, probably in its own commit.